### ISSUE TITLE
Intro and initial Reach out

### ISSUE THREAD

**Issue by ohgreatkate** [department-of-veterans-affairs/va.gov-team#65471]
Reach out to the team in the provided DSVA channel. 

Sample intro template:

:accessibility: [Menu item task: Product reviewed](link)
Hello [product team]! I will be your primary contact for the a11y design review. I will be following up through this thread for any questions, or additional resources I may need to get your work reviewed. I will be working through this over the next two weeks and provide you with updates along the way. What our design review will cover: [ADD DETAILS ON WHAT WILL BE REVIEWED, AND WHAT WILL NOT BE] Please do not hesitate to reach out to me as well for any questions or concerns.
Thank you! :corgi-wave:

**Comment by ohgreatkate** [comment]
Initial reach out and intro call completed along with weekly update. [Slack](https://dsva.slack.com/archives/CMNQT72LX/p1694627154763479)
